core/services/ocr2/plugins/mercury/integration_test.go

```diff
@@ -0,0 +1,501 @@
+	var (
+		oracles []confighelper.OracleIdentityExtra
+		nodes   []Node
+	)
+	// Set up n oracles
+
+	for i := int64(0); i < int64(n); i++ {
+		app, peerID, transmitter, kb := setupNode(t, bootstrapNodePort+i+1, fmt.Sprintf("oracle_mercury%d", i), []commontypes.BootstrapperLocator{
+			// Supply the bootstrap IP and port as a V2 peer address
+			{PeerID: bootstrapPeerID, Addrs: []string{fmt.Sprintf("127.0.0.1:%d", bootstrapNodePort)}},
+		}, backend, clientCSAKeys[i])
+
+		nodes = append(nodes, Node{
+			app, transmitter, kb,
+		})
+		offchainPublicKey, _ := hex.DecodeString(strings.TrimPrefix(kb.OnChainPublicKey(), "0x"))
+		oracles = append(oracles, confighelper.OracleIdentityExtra{
+			OracleIdentity: confighelper.OracleIdentity{
+				OnchainPublicKey:  offchainPublicKey,
+				TransmitAccount:   ocr2types.Account(hexutil.Encode(transmitter[:])),
+				OffchainPublicKey: kb.OffchainPublicKey(),
+				PeerID:            peerID,
+			},
+			ConfigEncryptionPublicKey: kb.ConfigEncryptionPublicKey(),
+		})
+	}
+
+	// Add the bootstrap job
+	// FIXME: If feedID is missing here, it boots and runs but does this:
+	//
+	//    logger.go:130: 2023-03-02T17:47:52.013-0500	INFO	bootstrap_mercury.EVM.Relayer	evm/config_poller.go:263	feed ID	{"feedID": "0x0000000000000000000000000000000000000000000000000000000000000000"}
+	//
+	// Why doesn't it error?
+	bootstrapNode.AddBootstrapJob(t, fmt.Sprintf(`
+type                              = "bootstrap"
+relay                             = "evm"
+schemaVersion                     = 1
+name                              = "boot"
+contractID                        = "%s"
+contractConfigTrackerPollInterval = "1s"
+
+[relayConfig]
+chainID = 1337
+feedID = "0x%x"
+	`, verifierAddress, feedID))
+
+	createBridge := func(name string, i, p int, borm bridges.ORM) (bridgeName string) {
+		bridge := httptest.NewServer(http.HandlerFunc(func(res http.ResponseWriter, req *http.Request) {
+			b, err := io.ReadAll(req.Body)
+			require.NoError(t, err)
+			require.Equal(t, `{"data":{"from":"ETH","to":"USD"}}`, string(b))
+			res.WriteHeader(http.StatusOK)
+			resp := fmt.Sprintf(`{"result": %d.0%d}`, p, i)
+			res.Write([]byte(resp))
+		}))
+		t.Cleanup(bridge.Close)
+		u, _ := url.Parse(bridge.URL)
+		bridgeName = fmt.Sprintf("bridge-%s-%d", name, i)
+		require.NoError(t, borm.CreateBridgeType(&bridges.BridgeType{
+			Name: bridges.BridgeName(bridgeName),
+			URL:  models.WebURL(*u),
+		}))
+
+		return bridgeName
+	}
+
+	// Add OCR jobs
+	for i, node := range nodes {
+		bmBridge := createBridge("benchmarkprice", i, 1568, node.App.BridgeORM())
+		askBridge := createBridge("ask", i, 1569, node.App.BridgeORM())
+		bidBridge := createBridge("bid", i, 1566, node.App.BridgeORM())
+
+		// create mercury job
+		node.AddJob(t, fmt.Sprintf(`
+type = "offchainreporting2"
+schemaVersion = 1
+name = "mercury-%[1]d"
+forwardingAllowed = false
+maxTaskDuration = "1s"
+contractID = "%[2]s"
+contractConfigTrackerPollInterval = "1s"
```

Comment:
@samsondav Is `1s` correct here? I tried to set it to 1s in my e2e tests but I got error `Error #01: contract config tracker poll interval must be between 15s and 2m0s, but is currently 1s`

Reply:
Yes it is correct, this integration test passes, not sure why you are getting this error.

Reply:
What is the stacktrace on that error?